src/System.Runtime/tests/System/IntPtr.cs

```diff
@@ -3,131 +3,192 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
-
+using System.Collections.Generic;
 using Xunit;
 
-public static unsafe class IntPtrTests
+public static class IntPtrTests
 {
+    private static unsafe bool s_is32Bits = sizeof(void*) == 4; // Skip IntPtr tests on 32-bit platforms
```

Comment:
(And for tests that obviously don't need to be omitted on 32-bit, you could let them run.)